controllers/utils/constants.go

```diff
@@ -137,3 +137,7 @@ const (
 	PlcHubTmplPrinfInNameErr = "printf variable is not supported in the template function Name field"
 	PlcHubTmplPrinfInNsErr   = "printf variable is not supported in the template function Namespace field"
 )
+
+// SpaceRequiredForPrecache is an env variable for precaching job that indicates the amount of space required
+// 							for precahing job
+const SpaceRequiredForPrecache = "15" // Gigabytes
```

Comment:
@vitus133 Should we use a different number if it's operator upgrade only? Or skip this step altogether?

Reply:
I think there is no typical number, because it depends on the user's use case.
I think we need to template this number sooner the better. It means we should accelerate extracting the precaching spec on the hub

Reply:
@imiller0 @browsell As we discussed, let's keep it simple and check the disk space against a static size for OCP upgrade. For operator upgrade, there isn't a good default to use. I imagine it would be a way smaller number. Is it small enough to be ignored?